src/main/java/com/pitchain/controller/OauthController.java

```diff
@@ -19,8 +19,11 @@
 public class OauthController {
     private final OauthService oauthService;
 
-    @Operation(summary = "소셜 로그인", description = "KAKAO: https://kauth.kakao.com/oauth/authorize?response_type=code&client_id=891063e7c569324189a0353dfb18c534&redirect_uri={redirectURL}을 통해 code값 받아오기, " +
-                                                    "NAVER: https://nid.naver.com/oauth2.0/authorize?response_type=code&client_id=T4z00JRW0P38KpqrIc6w&redirect_uri={redirectURL}을 통해 code값 받아오기")
+    @Operation(summary = "소셜 로그인", description = "각 Oauth 플랫폼의 endpoint를 통해 code값  조회 후 api 호출 <br>" +
+            "KAKAO: https://kauth.kakao.com/oauth/authorize?response_type=code&client_id=891063e7c569324189a0353dfb18c534&redirect_uri={redirectURL} <br> " +
+            "NAVER: https://nid.naver.com/oauth2.0/authorize?response_type=code&client_id=T4z00JRW0P38KpqrIc6w&redirect_uri={redirectURL} <br> " +
+            "GOOGLE: https://accounts.google.com/o/oauth2/v2/auth?client_id=936222247413-8cojftn1k2kipmrgspo9glprk5drrt3t.apps.googleusercontent.com&redirect_uri=http://localhost:8080&response_type=code&scope=email%20profile"
```

Comment:
```suggestion
            "GOOGLE: https://accounts.google.com/o/oauth2/v2/auth?client_id=936222247413-8cojftn1k2kipmrgspo9glprk5drrt3t.apps.googleusercontent.com&redirect_uri={redirectURL}&response_type=code&scope=email%20profile"
```
별 건 아니지만 그냥 통일성을 위해 ㅎㅎ...